mxd/backend-service/src/main/java/org/eclipse/mxd/entity/Content.java

```diff
@@ -0,0 +1,95 @@
+
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
Is this Copyright correct?

Reply:
I have took Copyright from eclipse-tractusx and changed in all files now.